src/components/case_manager/AllParas.tsx

```diff
@@ -6,22 +6,23 @@ import PersonCreationForm from "./PersonCreationForm";
 
 const AllParasPage = () => {
```

Comment:
Consider renaming this to MyParas (also, the Page suffix is not needed - it it technically a react component, and can in principle be used in other contexts)

Reply:
I've renamed it in this upcoming push.